docs/readme_chunks/generate_frontpage_documents.py

```diff
@@ -0,0 +1,112 @@
+import os
+import sys
+
+class DocGenerator:
+    """A Class to represent a documentation generator
+
+    Attributes: None
+
+    Methods
+    _________
+    get_chunk_paths()
+        returns the chunks that have been made
+    """
+
+
+    #CHUNKED DOCUMENTS
+    INTRO_DOC = "./docs/readme_chunks/introduction.md"
+    GET_INVOLVED = "./docs/readme_chunks/getinvolved.md"
+    ACC_STANDARDS = "./docs/readme_chunks/accessibility_standards.md"
+
+    # INTERNAL MESSAGES
+    DIR_ERR = "Error: please call from top directory, docs directory, or readme_chunks directory."
+
+    # PATHS TO WRITE TO
+    RTD_INDEX = "./docs/index.md"
+    REPO_README = "./README.md"
+
+
+    def __init__(self):
+        """
+        Constructs attributes needed for the object.
+        Additionally, parses chunk files and writes README files.
+        """
+        current_path = os.getcwd()
```

Comment:
Same from here onwards - I would suggest using Pathlib to leverage the use of `parent()` and the such